seqr/views/apis/dataset_api.py

```diff
@@ -112,6 +115,35 @@ def add_variants_dataset_handler(request, project_guid):
     Family.bulk_update(
         request.user, {'analysis_status': Family.ANALYSIS_STATUS_ANALYSIS_IN_PROGRESS}, guid__in=family_guids_to_update)
 
+    if project_has_analyst_access(project):
+        safe_post_to_slack(
+            SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL,
+            """{num_sample} new sample(s) are loaded in https://seqr.broadinstitute.org/project/{guid}/project_page
+            ```{samples}```
+            """.format(
+                num_sample=len(new_samples),
+                guid=project.guid,
+                samples=[sample.sample_id for sample in new_samples]
+            ))
+    elif project_has_anvil(project):
+        permission_levels = get_workspace_collaborator_perms(request.user, project.workspace_namespace, project.workspace_name)
+        send_html_email("""Hi {user},
+We are following up on your request to load data from AnVIL on {date}.
+We have loaded {num_sample} sample(s) from the AnVIL workspace <a>https://anvil.terra.bio/#workspaces/{namespace}/{name}</a> to the corresponding seqr project <a>https://seqr.broadinstitute.org/project/{guid}/project_page</a>. Let us know if you have any questions.
```

Comment:
It looks like we already have a constant for `https://anvil.terra.bio/` in communication_utils.py - we should move it to settings.py and re-use the constant here, rather than hardcoding the same URL in multiple places

Reply:
Should we get it from an environment variable?

Reply:
no I think hardcoding it is okay for now